docs/requirements.txt

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
I'd recommend cleaning this up by removing all the version constraints.  This will let you continue to use the file without adjustments in the future.

Reply:
So removed. I should note that requirements.txt is autogenerated, though: 

```
# This file is autogenerated by pip-compile with Python 3.9
# by the following command:
#
#    pip-compile docs/requirements.in
```